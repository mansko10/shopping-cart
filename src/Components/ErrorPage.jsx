import StyledErrorPage from "./Styles/StyledErrorPage";
import { Link } from "react-router-dom";

export default function ErrorPage() {
  return (
    <StyledErrorPage>
      <h1>This page doesn't exist</h1>
      <Link to="/">Go back to shop</Link>
    </StyledErrorPage>
  );
}
