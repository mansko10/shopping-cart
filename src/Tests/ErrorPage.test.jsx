import { it, expect } from "vitest";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import ErrorPage from "../Components/ErrorPage";

it("Renders ErrorPage.jsx correctly", () => {
  const { container } = render(<ErrorPage />, { wrapper: BrowserRouter });

  expect(container).toMatchSnapshot();
});
