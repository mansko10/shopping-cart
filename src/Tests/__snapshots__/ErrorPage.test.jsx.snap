// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Renders ErrorPage.jsx correctly 1`] = `
<div>
  <div
    class="sc-blHHSb gbkhKW"
  >
    <h1>
      This page doesn't exist
    </h1>
    <a
      data-discover="true"
      href="/"
    >
      Go back to shop
    </a>
  </div>
</div>
`;
